config/packages.yml

```
@@ -103,6 +103,10 @@ acquia/coding-standards:
   type: phpcodesniffer-standard
   url: ../coding-standards-php
 
+acquia/drupal-environment-detector:
+  type: package
```

Let's change this to the more standard "library", [which is Composer's default](https://getcomposer.org/doc/04-schema.md#type):

> Out of the box, Composer supports four types:
>
> - **library:** This is the default. It will copy the files to `vendor`.

```suggestion
  type: library
```

---

Thanks, I don't know why I chose `package` originally, maybe I didn't realize that it was based on Composer types 🤷 

I tested `library` in https://github.com/acquia/drupal-environment-detector/pull/15 and it seems to work.